src/object.c

```diff
@@ -1627,12 +1631,24 @@ NULL
         addReplyBulkCString(c,"allocator.resident");
         addReplyLongLong(c,server.cron_malloc_stats.allocator_resident);
 
+        addReplyBulkCString(c,"allocator.retained");
+        addReplyLongLong(c,server.cron_malloc_stats.allocator_retained);
+
+        addReplyBulkCString(c,"allocator.muzzy");
+        addReplyLongLong(c,server.cron_malloc_stats.allocator_muzzy);
+
         addReplyBulkCString(c,"allocator-fragmentation.ratio");
         addReplyDouble(c,mh->allocator_frag);
 
+        addReplyBulkCString(c,"allocator-fragmentation-smallbins.ratio");
+        addReplyDouble(c,mh->allocator_frag);
```

Comment:
`allocator_frag` was already printed (as allocator-fragmentation.ratio)
either we just replace that metric with a more accurate number, or if we want both, we need another variable.

Reply:
unify them.